.gitignore

```diff
@@ -105,7 +105,7 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
```

Comment:
this shouldn't make any difference (as per https://git-scm.com/docs/gitignore). A filename will apply at any level below. The file also appears lower down (which may be a dup)